src/components/CaptureFilter.js

```diff
@@ -367,22 +369,26 @@ function Filter(props) {
                 value={speciesId}
                 onChange={(e) => setSpeciesId(e.target.value)}
               >
-                {[
-                  { id: ALL_SPECIES, name: 'All' },
-                  {
-                    id: SPECIES_NOT_SET,
-                    name: 'Not set',
-                  },
-                  ...speciesContext.speciesList,
-                ].map((species) => (
-                  <MenuItem
-                    data-testid="species-item"
-                    key={species.id}
-                    value={species.id}
-                  >
-                    {species.name}
-                  </MenuItem>
-                ))}
+                {isLoading ? (
```

Comment:
In this file, the variable name `isLoading` is ambiguous.
When used in the code, it's not immediately obvious that it refers to _species_ loading, and could result in a name clash if we want to show loading indicators for any other fields.

Instead, just check `speciesContext.isLoading`.

Reply:
acknowledged and updated